client/src/partials/errors/404.html

```diff
@@ -0,0 +1,22 @@
+<div class="flex-content">
+  <div class="container-fluid" style="height: 100%;">
+    <div class="row">
+      <div class="col-md-4 col-md-offset-4">
+
+        <div class="panel panel-default content-error" data-error="404">
+          <div class="panel-heading clearfix text-center">
+            <strong>
+              <i class="fa fa-danger"></i>
+              <span class="text-danger">{{ "FORM.LABELS.ERROR_404" | translate }} </span>
+            </strong>
+          </div>
+
+          <div class="panel-body text-center">
+            <strong> {{"FORM.WARNINGS.PAGE_NOT_FOUND" | translate}}</strong>
+            <div style="background-image:radial-gradient(#CBCBCB, #FFFFFF); font-size:500%; color:white; text-shadow: 0px 0px 9px #777;"> 404 </div>
```

Comment:
Again this isn't directly relevant to this pull request (restructure of existing files). These styles are very hard coded into the templates themselves. It would be better for error pages to eventually use standard BHIMA classes that match the rest of the application.